src/pages/index.tsx

```diff
@@ -0,0 +1,415 @@
+                <div className="flex justify-end">
+                  <span>async/await</span> <ToggleSwitch />
+                </div>
+                {isToggled ? <QuickStartGuideWithAsyncAwait /> : <QuickStartGuide />}
+                <p>Finally, launch the server with:</p>
+                <CodeBlock className="language-bash">{'node server'}</CodeBlock>
+                <p>and you can test it with:</p>
+                <CodeBlock className="language-bash">{'curl http://localhost:3000'}</CodeBlock>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section className="section alternate">
+          <div className="container content">
+            <div className="columns is-centered">
+              <div className="column is-12">
+                <h1 className="title">Using CLI</h1>
+                <p>Get fastify-cli with NPM:</p>
+                <CodeBlock className="language-bash">{'npm install --global fastify-cli'}</CodeBlock>
+                <p>Then scaffold a new project with:</p>
+                Of course, Fastify can do much more than this.
+                <CodeBlock className="language-bash">{'fastify generate myproject'}</CodeBlock>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section className="section alternate">
+          <div className="container content">
+            <div className="columns is-centered">
+              <div className="column is-12">
+                <h1 className="title">Request/Response validation and hooks</h1>
+                <p>Of course, Fastify can do much more than this.</p>
+                <p>
+                  For example, you can easily provide input and output validation using JSON Schema and perform specific
+                  operations before the handler is executed:
+                </p>
+                <div className="flex justify-end">
+                  <span>async/await</span> <ToggleSwitch />
+                </div>
+                {isToggled ? <ReqResHooksWithAsyncAwait /> : <ReqResHooks />}
+              </div>
+            </div>
+          </div>
+        </section>
+        <section className="section alternate">
+          <div className="container content">
+            <div className="columns is-centered">
+              <div className="column is-12">
+                <h1 className="title">TypeScript Support</h1>
+                <p>
+                  Fastify is shipped with a typings file, but you may need to install
+                  <CustomHighLight text={'@types/node'} />, depending on the Node.js version you are using.
+                </p>
+                <p>The following example creates a http server.</p>
+                <p>We pass the relevant typings for our http version used.</p>
+                <p>By passing types we get correctly typed access to the underlying http objects in routes.</p>
+                <p>
+                  If using http2 we&39;d pass
```

Comment:
There are many (boring 😆) layout issues:

<img width="962" alt="image" src="https://user-images.githubusercontent.com/11404065/210860843-03ee3f5a-c404-4b6d-a274-db1559a71ae0.png">

Reply:
it was a typo...was required to write `&#39;` for apostrophes in html but `&39;` is all that was captured